sunrise-test-0.1/seeds.txt

```diff
@@ -0,0 +1 @@
+27d92a62f64585b995a6a99882cdd3a1a441336d@a.sunrise-test-1.cauchye.net:26656,9cb96bd137c6fab41446f5fa36e17b1f8896b05c@b.sunrise-test-1.cauchye.net:26656,db223ecc4fba0e7135ba782c0fd710580c5213a6@a-node.sunrise-test-1.cauchye.net:26656
```

Comment:
`89e5866f0c2edd5fd1cc604e0eb173d6111d9bd8` is current node id of validator-a

Reply:
Fixed